wbooks/src/app/components/AppNavigator/index.tsx

```diff
@@ -1,32 +1,39 @@
 import React from 'react';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { createStackNavigator } from '@react-navigation/stack';
+import SearchIcon from '@components/SearchIcon';
 import TabIcon from '@components/TabIcon';
 import colors from '@constants/colors';
 import SCREENS, { SCREEN_TITLES } from '@constants/screens';
 import { BottomTabParamList, LibraryStackParamList } from '@interfaces/navigatorParamLists';
 import BookDetails from '@screens/BookDetails';
 import Library from '@screens/Library';
+import SearchScreen from '@screens/SearchScreen';
 import WishList from '@screens/WishList';
 
-import customHeader from './config';
+import mainHeader, { searchHeader } from './config';
```

Comment:
Wrong!

Reply:
I refactored so I only have to import non-default imports.